apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/Logging/DefaultLogger.cs

```diff
@@ -94,8 +94,14 @@ protected override void LogPerformanceDataImpl()
         {
             while (true)
             {
-                await Task.Delay(PerformanceTraceLogInterval).ConfigureAwait(false);
-                LogPerformanceDataImpl();
+                await Task.Delay(PerformanceTraceLogInterval != TimeSpan.Zero ?
```

Comment:
I've figured out (I think!) why this is doing this, but perhaps it's worth adding an explanatory comment? (It may be clearer already when looking at all the code together rather than doing a code review, of course.)